README.md

```diff
@@ -55,6 +54,45 @@ release.
 
 An example of such `release.yml` can be found at: https://github.com/snowdrop/spring-boot-bom/blob/sb-2.3.x/release.yml
 
+### Init release repositories for a new Major.Minor
+
+Whenever a new Major.Minor release occurs:  
+* a new `sb-<major>.<minor>.x` branch must be created in the `snowdrop/spring-boot-bom` github repostory 
+* a new folder in the `middleware/build-configurations` gitlab repository must be created.
+
+This command performs these operations automatically.
+
+> NOTE: The previous release is used as a source of information. It is used as the source for the new 
+`snowdrop/spring-boot-bom` github repostory branch and as the source for the new `build-configurations`
+configuration.
+
+```bash
+java -jar target/issues-manager-$(xpath -q -e  "/project/version/text()" pom.xml)-runner.jar \
+    -u <jira user> -p <jira password> 
+    new-maj-min 
+    -g <github org>/<github repo>/<git reference: branch, tag, hash> \ 
+    -o <github token> \
+    -glu <gitlab user> \
+    -glt <gitlab token> \ 
+    -r <new release> \
+    -pr <previous release>
+```
+
+e.g.
+
+```bash
+java -jar target/issues-manager-$(xpath -q -e  "/project/version/text()" pom.xml)-runner.jar \
```

Comment:
Fix the command as mentioned :up:

Reply:
It should be fixed by rebasing the branch onto main one

Reply:
fixed